src/interfaces/store-interfaces/state-interface.ts

```diff
@@ -1,9 +1,8 @@
 import {
   AccountInformationIF, CertifyIF, DefineCompanyIF, NameRequestIF, TombStoneIF, PeopleAndRolesIF,
   ShareStructureIF, DateTimeIF, IncorporationAgreementIF, BusinessInformationIF, IncorporationFilingIF,
-  FilingDataIF
+  FilingDataIF, StaffPaymentStepIF
 } from '@/interfaces'
```

Comment:
Maybe we should simplify this import as I've done elsewhere using `*` and `...` :)

Reply:
Will implement it in the next PR